Add Docker File and Docker compose.

jeanvetorello opened the pull request:
add Docker File and docker-compose 

Pessoal estou adicionando um docker file e um docker compose para ajudar o pessoal a utilizar varias contas, ao inves de utilizar maquinas virtuais, e tambem para  facilitar o uso em vps.

para rodar bastar ter docker e docker compose instalado e ajustar o - VNC_PASSWORD=COLOCARUMASENHAAQUI.  depois docker-compose up -d --build
essa image já tem o vnc instalado entao depois que ela rodar  entre com o vnc no 127.0.0.1:5900 , coloca a senha que está la na variavel e pronto. vc está acessando remoto um linux,

apos isso abre o firefox instale metamask , add a rede da bsc e pronto.
vai ter uma pasta com o repositorio atualizado no /root/Desktop

OBS: essa ultima atualizaçao do requirements.txt quebrou o build do container entao retirar o simpleaudio pois não tem audio mesmo .

TODO: acrescentar um volume onde guarde as info do firefox para que não se perca quando o continaer reinicar.

jeanvetorello committed:
Add Docker File and Docker compose.

add Docker File and docker-compose 

Pessoal estou adicionando um docker file e um docker compose para ajudar o pessoal a utilizar varias contas, ao inves de utilizar maquinas virtuais, e tambem para  facilitar o uso em vps.

para rodar bastar ter docker e docker compose instalado e ajustar o - VNC_PASSWORD=COLOCARUMASENHAAQUI.  depois docker-compose up -d --build
essa image já tem o vnc instalado entao depois que ela rodar  entre com o vnc no 127.0.0.1:5900 , coloca a senha que está la na variavel e pronto. vc está acessando remoto um linux,

apos isso abre o firefox instale metamask , add a rede da bsc e pronto.
vai ter uma pasta com o repositorio atualizado no /root/Desktop

OBS: essa ultima atualizaçao do requirements.txt quebrou o build do container entao retirar o simpleaudio pois não tem audio mesmo .

TODO: acrescentar um volume onde guarde as info do firefox para que não se perca quando o continaer reinicar.

jeanvetorello committed:
Create secrets.py

jeanvetorello committed:
Update .gitignore

jeanvetorello committed:
Update index.py

jeanvetorello committed:
Update index.py

fguide commented:
Boa ideia, seria interessante adicionar na imagem também o node.js para ser possível configurar o NoVNC e fazer o acesso remoto mais simples por HTML5

Um exemplo:
https://github.com/theasp/docker-novnc

itxtoledo commented on index.py:
```diff
@@ -69,7 +70,13 @@
 last_log_is_progress = False
 
-
+def telegram_bot_sendtext(bot_message):
+    bot_token = secrets.bot_token
+    bot_chatID = secrets.bot_chatID
+    conta = secrets.conta
+    send_text = 'https://api.telegram.org/bot' + bot_token + '/sendMessage?chat_id=' + bot_chatID + '&parse_mode=Markdown&text=' + 'Conta: '+ conta + '\n' + bot_message
+    response = requests.get(send_text)
+    return response.json()
```
Isso não tem a ver com a alteração principal.

jeanvetorello replied:
apa desculpe tinha feito essas alteração  so pra min, oque vale so so o docker file e o docker compose comites depois vou remover

itxtoledo commented on index.py:
```diff
@@ -267,9 +274,9 @@ def alertCaptcha():
     if len(popup_pos) == 0:
         return "not-found"
-
+    tele = telegram_bot_sendtext("⚠️ ATENÇÃO! \n\n 🧩 RESOLVER NOVO CAPTCHA!")
```
Você alterou outras partes do código que não tem a ver com a adição do docker.

itxtoledo reviewed (changes requested):
Por favor retire as alterações que não tem a ver com o docker, isso deve ser evitado pois deixa o git blame bagunçado, e não desative features que são necessárias.

jeanvetorello committed:
Revert "Update index.py"

This reverts commit ab0875425d8b69080d2364ddcb1a78d7df0fcbf6.

jeanvetorello committed:
Revert "Revert "Update index.py""

This reverts commit a34d74de624d5b64e736e8bb522ccbd633a11d96.

jeanvetorello committed:
Revert "Add Docker File and Docker compose."

This reverts commit d0d6a416a94a9b50dfcf30cac456aafa379adff7.

jeanvetorello committed:
Revert "Create secrets.py"

This reverts commit b03d09ad9d14723d39e078a7150709bc35c36882.

jeanvetorello committed:
Revert "Revert "Add Docker File and Docker compose.""

This reverts commit fea9d7f7c0f20361ce72188198177a5bd545746d.

jeanvetorello committed:
Update docker-compose.yml

ajuste para acessar o container por http

para acesso o container por http abrir o browser e acessar localhost:6080  user root senha que mudou no HTTP_PASSWORD

jeanvetorello commented:
> Boa ideia, seria interessante adicionar na imagem também o NoVNC e fazer o acesso remoto mais simples por HTML5
> 
> Um exemplo: https://github.com/theasp/docker-novnc

feito no ultimo commit

lucasvieceli commented:
isso ja n deveria estar no branch main ??

estou procurando e não encontro
